x-pack/plugins/canvas/canvas_plugin_src/functions/common/dropdown_control.test.ts

```diff
@@ -18,14 +18,14 @@ describe('dropdownControl', () => {
     expect(
       fn(
         testTable,
-        { filterColumn: 'name', valueColumn: 'name' },
+        { filterColumn: 'name', valueColumn: 'name', labelColumn: '', filterGroup: '' },
```

Comment:
Same question here? Why the need to add filterGroup?

Reply:
The same, as above.

Reply:
Done.